dist/s_typedef

```diff
@@ -32,8 +32,9 @@ build() {
         echo "    typedef $t $n $upper;"
     done
 
-    # There's one fixed type we use.
+    # Following fixed types we use.
```

Comment:
Nitpick: grammar here is a bit weird, how about just "Fixed types we use."?

Reply:
Fixed as suggested.